javascript/javascript2/week3/JS2W3.js

```diff
@@ -0,0 +1,116 @@
+///////////////////////////////////////////////////////Function as a variable
+
+//Create an array with 3 items. All items should be functions. 
+//Iterate through the array and call all the functions.
+
+const arr = [function0, function1, function2];
+
+function function0() {
+    console.log('I am function0');
+};
+
+function function1() {
+    console.log('I am function1');
+};
+
+function function2() {
+    console.log('I am function2');
+};
+arr.forEach(functionList => functionList());
+
+/*Create a function as a const and try creating a function normally. 
+Call both functions. */
+
+//functionCreated(); //It gives an error if you call it by function name (func1),
+//Moreover it can not be done here before declaration however the normal function can be done 
+//before everything in the program 
+
+normalFunc();
+
+const functionCreated = function func1() {
+    console.log('I am a function attached with a constant');
+};
+
+function normalFunc() {
+    console.log('I am a normal function');
+};
+
+functionCreated(); //It gives an error if you call it by function name (func1)
+normalFunc();
+
+//Create an object that has a key whose value is a function.
+//Try calling this function.
+
+function functionObject() {
+    console.log('Call me, I am in the object');
+};
+object = {
+    'key': functionObject
```

Comment:
Try to not use "object" as a name.
you can use the arrow function here as well like this:

const objectWithFunctionKey = {
  functionKey: () => console.log("object"),
};